samples/detect.js

```diff
@@ -23,24 +23,40 @@ const {Transform, pipeline} = require('stream');
 const pump = util.promisify(pipeline);
 
 function detectTextIntent(projectId, sessionId, queries, languageCode) {
+  const PROJECT_ID = projectId;
```

Comment:
These 4 variables shouldn't be defined here.  They're supposed to come in as function arguments, right?

Reply:
Yes, those 4 lines should be deleted. We show them as a comment if the `region tag` starts after the function signature - which is exactly what you have below.

Reply:
Well, the problem here is that the variables that come as parameters (`projectId`, `sessionId`, `queries`, and `languageCode`) are not shown in the documentation because they are only part of the wrapper function.
What the end user will see in the docs is the code wrapped between the brackets
```
// [START dialogflow_detect_intent_text]
...
// [END dialogflow_detect_intent_text]
```

I was told that we would like the user to be able to simply copy/paste the code and run it. If we reference variables that are not defined the code will clearly fail when the user tries to run it.

Thoughts?

Reply:
If we delete them then I can update the commented variables to match the names from the parameters. Something like this:

```
function detectTextIntent(projectId, sessionId, queries, languageCode) {
  // [START dialogflow_detect_intent_text]

  /**
   * TODO(developer): UPDATE these variables before running the sample.
   */
  // const projectId = 'PROJECT_ID';
  // const sessionId = 123456;  // Random number or hashed user identifier
  // const queries = [
  //   'Reserve a meeting room in Toronto office, there will be 5 of us',
  //   'Next monday at 3pm for 1 hour, please', // Tell the bot when the meeting is taking place
  //   'B'  // Rooms are defined on the Dialogflow agent, default options are A, B, or C
  // ]
  // const languageCode = 'en';
...
```

Thoughts?

Reply:
Ok @JustinBeckwith,
Perfect, I will update it to be like this (see below). If that's ok with you I can push the changes.

```
function detectTextIntent(projectId, sessionId, queries, languageCode) {
  // [START dialogflow_detect_intent_text]

  /**
   * TODO(developer): UPDATE these variables before running the sample.
   */
  // projectId: ID of the GCP project where Dialogflow agent is deployed
  // const projectId = 'PROJECT_ID';
  // sessionId: Random number or hashed user identifier
  // const sessionId = 123456;
  // queries: A set of sequential queries to be send to Dialogflow agent for Intent Detection 
  // const queries = [
  //   'Reserve a meeting room in Toronto office, there will be 5 of us',
  //   'Next monday at 3pm for 1 hour, please', // Tell the bot when the meeting is taking place
  //   'B'  // Rooms are defined on the Dialogflow agent, default options are A, B, or C
  // ]
  // languaceCode: Indicates the language Dialogflow agent should use to detect intents
  // const languageCode = 'en';
...
```

Reply:
Or maybe this way:


```
function detectTextIntent(projectId, sessionId, queries, languageCode) {
  // [START dialogflow_detect_intent_text]

  /**
   * TODO(developer): UPDATE these variables before running the sample.
   * projectId: ID of the GCP project where Dialogflow agent is deployed.
   * sessionId: Random number or hashed user identifier.
   * queries: A set of sequential queries to be send to Dialogflow agent for Intent Detection.
   * languaceCode: Indicates the language Dialogflow agent should use to detect intents.
   */
  // const projectId = 'PROJECT_ID';
  // const sessionId = 123456;
  // const queries = [
  //   'Reserve a meeting room in Toronto office, there will be 5 of us',
  //   'Next monday at 3pm for 1 hour, please', // Tell the bot when the meeting is taking place
  //   'B'  // Rooms are defined on the Dialogflow agent, default options are A, B, or C
  // ]
  // const languageCode = 'en';
...
```

Reply:
The first one above looks more correct :)

Reply:
Done.